src/App.js

```diff
@@ -1,22 +1,24 @@
 import React from 'react';
 import './App.scss';
+import { TodoList } from './components/TodoList';
 
 import todos from './api/todos';
 import users from './api/users';
 
+function findUser(userId) {
+  return users.find(user => userId);
+}
+
+const preparedTodos = todos.map(todo => ({
+  ...todo,
+  user: findUser(todo.userId),
+}));
```

Comment:
I think findUser is unnecessary here

```suggestion
const preparedTodos = todos.map(todo => ({
  ...todo,
  user: users.find(user => todo.userId),
}));
```